parlai/core/torch_agent.py

```diff
@@ -821,7 +972,7 @@ def _set_label_cands_vec(self, obs, add_start, add_end, truncate):
                 for c in obs['label_candidates']]
         return obs
 
-    def vectorize(self, obs, add_start=True, add_end=True, split_lines=False,
+    def vectorize(self, obs, history, add_start=True, add_end=True,
```

Comment:
Why have the history passed in as a parameter, instead of just using self.history?

Reply:
I guess the same reason we pass in observation instead of using self.observation (easier abstraction???)? thoughts @alexholdenmiller?

Reply:
Good enough for me